app/src/utils/MapEditControls.tsx

```diff
@@ -126,7 +126,7 @@ const MapEditControls: React.FC<IMapEditControlsProps> = (props) => {
     const { onMounted } = props;
 
     onMounted && onMounted(drawRef.current);
-  }, [props.draw, props.edit, props.position]);
+  }, []);
```

Comment:
these dependencies need to stay:

```suggestion
  }, [props.draw, props.edit, props.position]);
```

Reply:
there should be no changes to this file I believe :)

Reply:
ah ok